### PR TITLE
add support for PRISMA_*_BINARY

### PR DESCRIPTION
**Pull request by sergioramos**

### CODE REVIEW

**Comment by yuramedvedev** [comment]
It would be great to have this feature merged!

**Comment by timsuchanek** [comment]
@sergioramos please sign the CLA and could you summarize what the PR is trying to accomplish. 🙏

**Comment by janpio** [comment]
ping @sergioramos

**Comment by sergioramos** [comment]
I was trying to fix https://github.com/prisma/prisma2/issues/1034 because I also stumbled upon it. going to close this PR and re-open if it is still happening

**Comment by janpio** [comment]
Thanks @sergioramos!
